build.sh

```diff
@@ -17,6 +17,7 @@ debug_opts='--enable-debug'
 user_opts=''
 build_cflags='-O0 -g'
 job_count='4'
+make_opts='make'
```

Comment:
`make_opts` sounds misleading for this, because this is just the name of the make-command, and not just the options for `make`.

Reply:
FreeBSD uses gmake tool

Reply:
I think what he's suggesting is the variable name is slightly misleading. `make_opts` would imply that you're changing some options *for* make not *what* you're using to compile.

Reply:
I changed name variable

Reply:
@GXTX  ?

Reply:
What?

Reply:
@GXTX do you have next comments for this PR ?